rust_src/src/buffers.rs

```diff
@@ -4,9 +4,11 @@ use libc::{c_int, c_uchar, c_void, ptrdiff_t};
 use std::{mem, ptr};
 
 use remacs_macros::lisp_fn;
-use remacs_sys::{EmacsInt, Lisp_Buffer, Lisp_Object, Lisp_Overlay, Lisp_Type, Vbuffer_alist};
-use remacs_sys::{make_lisp_ptr, nsberror, set_buffer_internal};
+use remacs_sys::{globals, EmacsInt, Lisp_Buffer, Lisp_Object, Lisp_Overlay, Lisp_Type,
+                 Qbuffer_read_only, Qinhibit_read_only, Qnil, Vbuffer_alist};
```

Comment:
We have been putting the Qfoo and Vfoo items in their own import statements to help minimize conflicts.